### ISSUE TITLE
test CKV_AWS_101: "Ensure Neptune logging is enabled" fails when set correctly

### ISSUE THREAD

**Issue by JamesWoolfenden** [bridgecrewio/checkov#856]
Incorrectly failing test:
Even though set correctly this rule is triggered
![image](https://user-images.githubusercontent.com/1456880/107698106-f88b0580-6cab-11eb-8962-87c96ff43d2e.png)

If you set it to be a string the test passes:
  enable_cloudwatch_logs_exports      = "audit"

But this attribute has to be a set of string.

**Comment by nimrodkor** [comment]
@JamesWoolfenden That's weird considering the test case [here](https://github.com/bridgecrewio/checkov/blob/2522f0bca5848c03e56acc41a81d2914f83aa7e3/tests/terraform/checks/resource/aws/test_NeptuneClusterLogging.py#L35)
Mind sharing it as a string?

**Comment by JamesWoolfenden** [comment]
i know its odd, amyway i can do better than that heres a module that demos it https://github.com/JamesWoolfenden/terraform-aws-neptune

**Comment by JamesWoolfenden** [comment]
version: 1.0.775

**Comment by nimrodkor** [comment]
That's weird. I fixed a bug with this check a few versions ago, and I can't reproduce your issue. 
For version 1.0.735, it was failing. For version 1.0.775, It passes for me. The outputs:

**1.0.735**:
```
       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 1.0.735 
Update available 1.0.735 → 1.0.775
Run pip3 install -U checkov to update 


terraform scan results:

Passed checks: 3, Failed checks: 1, Skipped checks: 0

Check: CKV_AWS_102: "Ensure Neptune Cluster instance is not publicly available"
        PASSED for resource: aws_neptune_cluster_instance.example
        File: /aws_neptune_cluster_instance.example.tf:1-8
        Variable common_tags (of variables.tf) evaluated to value "{'createdby': 'Terraform'}" in expression: common_tags = ${var.common_tags}

Check: CKV_AWS_44: "Ensure Neptune storage is securely encrypted"
        PASSED for resource: aws_neptune_cluster.default
        File: /aws_neptune_cluster.default.tf:1-13
        Variable common_tags (of variables.tf) evaluated to value "{'createdby': 'Terraform'}" in expression: common_tags = ${var.common_tags}
        Guide: https://docs.bridgecrew.io/docs/general_18

Check: CKV_AWS_41: "Ensure no hard coded AWS access key and secret key exists in provider"
        PASSED for resource: aws.default
        File: /example/examplea/provider.aws.tf:1-3
        Guide: https://docs.bridgecrew.io/docs/bc_aws_secrets_5

Check: CKV_AWS_101: "Ensure Neptune logging is enabled"
        FAILED for resource: aws_neptune_cluster.default
        File: /aws_neptune_cluster.default.tf:1-13

                1  | resource "aws_neptune_cluster" "default" {
                2  |   cluster_identifier                  = var.cluster["cluster_identifier"]
                3  |   engine                              = var.cluster["engine"]
                4  |   backup_retention_period             = var.cluster["backup_retention_period"]
                5  |   preferred_backup_window             = var.cluster["preferred_backup_window"]
                6  |   skip_final_snapshot                 = var.cluster["skip_final_snapshot"]
                7  |   iam_database_authentication_enabled = var.cluster["iam_database_authentication_enabled"]
                8  |   apply_immediately                   = var.cluster["apply_immediately"]
                9  |   storage_encrypted                   = true
                10 |   neptune_subnet_group_name           = aws_neptune_subnet_group.default.name
                11 |   enable_cloudwatch_logs_exports      = ["audit"]
                12 |   tags                                = var.common_tags
                13 | }

        Variable common_tags (of variables.tf) evaluated to value "{'createdby': 'Terraform'}" in expression: common_tags = ${var.common_tags}
```

**1.0.775**:
```
       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 1.0.775 

terraform scan results:

Passed checks: 4, Failed checks: 0, Skipped checks: 0

Check: CKV_AWS_102: "Ensure Neptune Cluster instance is not publicly available"
        PASSED for resource: aws_neptune_cluster_instance.example
        File: /aws_neptune_cluster_instance.example.tf:1-8
        Variable common_tags (of variables.tf) evaluated to value "{'createdby': 'Terraform'}" in expression: common_tags = ${var.common_tags}

Check: CKV_AWS_101: "Ensure Neptune logging is enabled"
        PASSED for resource: aws_neptune_cluster.default
        File: /aws_neptune_cluster.default.tf:1-13
        Variable common_tags (of variables.tf) evaluated to value "{'createdby': 'Terraform'}" in expression: common_tags = ${var.common_tags}

Check: CKV_AWS_44: "Ensure Neptune storage is securely encrypted"
        PASSED for resource: aws_neptune_cluster.default
        File: /aws_neptune_cluster.default.tf:1-13
        Variable common_tags (of variables.tf) evaluated to value "{'createdby': 'Terraform'}" in expression: common_tags = ${var.common_tags}
        Guide: https://docs.bridgecrew.io/docs/general_18

Check: CKV_AWS_41: "Ensure no hard coded AWS access key and secret key exists in provider"
        PASSED for resource: aws.default
        File: /example/examplea/provider.aws.tf:1-3
        Guide: https://docs.bridgecrew.io/docs/bc_aws_secrets_5
```

**Comment by JamesWoolfenden** [comment]
Can confirm 1.0.775 works for me
